packages/node_modules/@webex/internal-plugin-llm/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@webex/internal-plugin-voicea",
+    "version": "2.19.2",
+    "description": "",
+    "license": "MIT",
+    "main": "dist/index.js",
+    "devMain": "src/index.js",
+    "repository": "https://github.com/webex/webex-js-sdk/tree/master/packages/node_modules/@webex/internal-plugin-voicea",
+    "engines": {
+      "node": ">=8"
+    },
+    "browserify": {
+      "transform": [
+        "babelify",
+        "envify"
+      ]
+    }
```

Comment:
can you add the dependency here, it will work if you dont add it but need it for reference 

mainly mercury and core would be the one i guess

Reply:
added